docs/notebooks/dispersion/stix_dispersion.ipynb

```diff
@@ -0,0 +1,768 @@
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import scipy\n",
+    "\n",
+    "from astropy import units as u\n",
+    "from astropy.constants import eps0\n",
+    "from astropy.constants.si import c\n",
+    "\n",
+    "from plasmapy.dispersion.analytical.stix_ import stix\n",
+    "from plasmapy.dispersion.analytical.two_fluid_ import two_fluid\n",
+    "from plasmapy.formulary import speeds\n",
+    "from plasmapy.formulary.frequencies import gyrofrequency, plasma_frequency\n",
+    "from plasmapy.formulary.lengths import inertial_length\n",
+    "from plasmapy.particles import Particle\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = [10.5, 0.56 * 10.5]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f5d0b2fc",
+   "metadata": {},
+   "source": [
+    "## Wave Normal to the Surface\n",
+    "\n",
+    "To calculate the normal surface waves propagating through a magnetized uniform cold plasma. The wave which is normal to the surface, is the locus of the phase velocity $\\textbf{v}_{phase} = \\frac{\\omega}{k} \\, \\hat{k}$ where $\\hat{k} = \\frac{\\textbf{k}}{k}$. The equation for the wave normal surface can be derived via the prior equations, resulting in the form of\n",
+    "\n",
+    "$$\n",
+    "    Au^{4} + Bu^{2} + C = 0\n",
+    "$$\n",
+    "\n",
+    "where $u = \\frac{\\omega}{ck}$. To begin we define the required parameters to compute the wave numbers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ab1a4b91",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define input parameters\n",
+    "inputs_1 = {\n",
+    "    \"theta\": np.linspace(0, np.pi, 50) * u.rad,\n",
+    "    \"ions\": Particle(\"p\"),\n",
+    "    \"n_i\": 1e12 * u.cm**-3,\n",
+    "    \"B\": 0.43463483142776164 * u.T,\n",
+    "    \"w\": 41632.94534008216 * u.rad / u.s,\n",
+    "}\n",
+    "\n",
+    "# define a meshgrid based on the number of theta values\n",
+    "omegas, thetas = np.meshgrid(\n",
+    "    inputs_1[\"w\"].value, inputs_1[\"theta\"].value, indexing=\"ij\"\n",
+    ")\n",
+    "omegas = np.dstack((omegas,) * 4).squeeze()\n",
+    "thetas = np.dstack((thetas,) * 4).squeeze()\n",
+    "\n",
+    "# compute k values\n",
+    "k = stix(**inputs_1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57d74632",
+   "metadata": {},
+   "source": [
+    "[Quantity]: https://docs.astropy.org/en/stable/api/astropy.units.Quantity.html#astropy.units.Quantity\n",
+    "\n",
+    "The computed wavenumbers in units (rad/m) are returned in a dictionary (shape $N \\times M \\times 4$), with the keys representing $\\theta$ and the values (instances of Astropy [Quantity]) being the wavenumbers. The first dimension maps to the $w$ array, the second dimension maps to the $\\theta$ array, and the third dimension maps to the four roots of the Stix polynomial.\n",
+    "* $k[0]$ is the square root of the positive quadratic solution\n",
```

Comment:
```suggestion
    "\n",
    "* $k[0]$ is the square root of the positive quadratic solution\n",
```
This might be necessary to make the bullet point list render correctly.